en/middleware/micrortps.md

```diff
@@ -131,16 +165,137 @@ $ ./micrortps_agent [options]
 
 To launch the *Agent*, run `micrortps_agent` with appropriate options for specifying the connection to the *Client* (the default options connect from a Linux device to the *Client* over a UART port).
 
+As an example, to start the *micrortps_agent* with connection through UDP, issue:
+
+```sh
+./micrortps_agent -t UDP
+```
+
+## Agent interfacing with a ROS2 middleware
+
+`px4_ros_com` build structure allows to automatically generate and build the agent application. Since it is also installed using the [`colcon`](http://design.ros2.org/articles/build_tool.html) build tools, running it works exactly the same way as the above. Check the **Building the `px4_ros_com` package** for details about the build structure.
+
+
+## Building the `px4_ros_com` package
```

Comment:
Add anchor by appending to end of heading line: `{#px4_ros_com}`

Reply:
right makes sense.

Reply:
I wasn't actually able to make this work. Can be addressed later I guess.